### PR TITLE
Revert "[dashboard] Downtime in-app announcements (#18761)"

### PR DESCRIPTION
**Pull request by geropl**
This reverts commit cebee56ff834a0017c42a799c298ef0e8df67db7.

## Description

<details>
<summary>Summary generated by Copilot</summary>

### <samp>🤖 Generated by Copilot at 1a5845a</samp>

Removed maintenance notifications and related code from the dashboard. Simplified the `Alert` and `AppRoutes` components.

</details>

## Related Issue(s)

Fixes EXP-684

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/components/Alert.tsx [hunk 1]
```diff
@@ -123,14 +122,14 @@ export default function Alert(props: AlertProps) {
...
-                        className="bg-transparent hover:bg-transparent"
+                    <button
+                        type="button"
+                        className="bg-transparent p-1"
                         onClick={handleClose}
                         autoFocus={autoFocusClose}
                     >
                         <XSvg className="w-3 h-4 cursor-pointer" />
-                    </Button>
+                    </button>
```
Comment (gtsiolis) [inline components/dashboard/src/components/Alert.tsx:132]: **thought**: We might want to keep these changes around, but should be fine to revert in this PR as we only use the closable prop in two instances, see [relevant diff](https://github.com/search?q=repo%3Agitpod-io%2Fgitpod+closable%3D%7Btrue%7D+&type=code). 🏓

**Review by gtsiolis — APPROVED** [review]
Deleted code is the best code. Thanks, @geropl. 🌮

File: components/dashboard/src/components/Alert.tsx (hunk 1 above)
Comment (geropl) [reply components/dashboard/src/components/Alert.tsx:132]: Let's keep them right away!

**Pull request merged**